cheat_sheets/pydpf-core_cheat_sheet/pydpf-core_cheat_sheet.tex

```diff
@@ -0,0 +1,203 @@
+\documentclass[9pt,landscape]{article}
+\usepackage{{./../static/style}}
+\pdfinfo{
+  /Title (PyDPF-Core Cheat Sheet)
+  /Creator (TeX)
+  /Producer (pdfTeX 1.40.0)
+  /Author (Ansys)
+  /Subject (PyDPF)
+  /Keywords (PyAnsys, PyDPF, DPF, Cheat sheet, template)}
+
+\begin{document}
+\raggedright
+\footnotesize
+
+% Add the title of cheat sheet here
+% ----------------------------------------
+
+\begin{center}
+     \Huge{\textbf{Cheat sheet for PyDPF-Core}} \\
+\end{center}
```

Comment:
it would be great if you can add the version here (example)
\begin{center}
\small{\textbf{version:0.6.78 (stable) }} \\
\end{center}

Reply:
@Revathyvenugopal162 oh so the Cheatsheet should be linked to a specific package version? Or the Cheatsheet itself should be versioned?
I understand versioning the Cheatsheet, but not so much linking it to a package version.

Reply:
Hi @PProfizi!

Adding the version serves as a reference for the PyDPF Post package version used to build the cheat sheet. That way, if for some reason the API changes we can try to adapt it from time to time.

Also, @Revathyvenugopal162, maybe the message shold be something like this:

```tex
\begin{center}
\small{\textbf{based on version:0.6.78 (stable) - API may be subjected to change}} \
\end{center}
```

Reply:
No problem! We are always improving 😄

Reply:
Added!